pulp_rpm/app/serializers.py

```diff
@@ -250,7 +250,8 @@ def deferred_validate(self, data):
         except OSError:
             raise NotAcceptable(detail='RPM file cannot be parsed for metadata.')
 
-        package = Package.objects.filter(**new_pkg)
+        attrs = {key: new_pkg[key] for key in Package.natural_key_fields()}
```

Comment:
Sorry I should have mentioned in my last comment that this dict is available also as natural_key_dict https://github.com/pulp/pulpcore/blob/9aac516c851b6fce68dcc620b2d4fba5a4679d19/pulpcore/app/models/content.py#L279

Reply:
Oh wait maybe this is right because it pulls them from `new_pkg`